File: PlayMinds-Back/src/main/java/com/coderhouse/pmb/Entitys/Game.java
package com.coderhouse.pmb.Entitys;
import jakarta.persistence.*;
import lombok.Data;

import java.io.Serializable;

import com.coderhouse.pmb.Enum.TypeGame;

@Data
@Entity
@Table(name = "game")
public class Game implements Serializable {
    @Id
    private String id_game;
    private String name_game;

    @Enumerated(EnumType.STRING)
    private TypeGame type_game;
    private String description;
    private int value_points;

    @ManyToOne
    @JoinColumn(name = "FK_user")
    private User user;
}

File: play-minds/src/models/Entitys/Game.ts
import { Request } from "@/helpers/requests";
import { User } from "./User";

export class Game {
  id_game?: string;
  name_game: string;
  type_game: string;
  description: string;
  value_points: number;
  user: User;

  constructor(
    id_game: string,
    name_game: string,
    type_game: string,
    description: string,
    value_points: number,
    user: User
  ) {
    this.id_game = id_game;
    this.name_game = name_game;
    this.type_game = type_game;
    this.description = description;
    this.value_points = value_points;
    this.user = user;
  }
}
